h/search/persistence.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+"""Code for persisting (indexing) annotations to Elasticsearch."""
+
+from __future__ import unicode_literals
+
+import elasticsearch_dsl
+
+
+class Annotation(elasticsearch_dsl.DocType):
+    """An elasticsearch_dsl annotation document."""
+
+    class Meta:
+        index = "hypothesis"
```

Comment:
The only place where we use any index name other than `"hypothesis"` is in the tests and those hardcode the index name. The `"es.index"` / `ELASTICSEARCH_INDEX` setting is not used and can be deleted.

Reply:
By "not used" you mean that we don't ever _set_ it in any of our dev/prod environments? The `es.index` setting is clearly used in the sense it gets read by `h.search.client.get_client`.

Reply:
I don't think it ever comes from a config file or env var